subroutines.sh

```diff
@@ -119,6 +150,7 @@ listen_to_ble() {
   while :; do
     bluetoothctl_read
 
+    BLTCTL_COMMAND_DEVICES=false
     for position in $(seq $n_vins); do
```

Comment:
hmmm I suspect this will raise shellcheck issues?

Reply:
It is defined in env.sh and we load that first.